slam-libraries/viam-cartographer/src/io/read_PCD_file.h

```diff
@@ -15,16 +15,16 @@
 namespace viam {
 namespace io {
 static const int filenamePrefixLength = 6;
-class ReadFile {
-   public:
-    std::vector<std::string> listFilesInDirectory(std::string data_directory);
-    cartographer::sensor::TimedPointCloudData timedPointCloudDataFromPCDBuilder(
-        std::string file_path, std::string initial_filename);
-    int removeFile(std::string);
-
-    // Converts UTC time string to a double value.
-    double ReadTimeFromFilename(std::string filename);
-};
+
+std::vector<std::string> ListFilesInDirectory(std::string data_directory);
+
+cartographer::sensor::TimedPointCloudData TimedPointCloudDataFromPCDBuilder(
+    std::string file_path, double start_time);
+
+int RemoveFile(std::string);
+
+// Converts UTC time string to a double value.
+double ReadTimeFromFilename(std::string filename);
 
```

Comment:
Decided to take functions out of the class because they don't really share any variables and are just a collection of functions. Easier to understand where they're coming from since we now have to simply use the namespace `viam::io::FUNCTION`